### DISCUSSION TITLE
foobar2000 v2.0 64 bits support

### DISCUSSION THREAD

**Discussion by chris68790** [TheQwertiest/foo_spider_monkey_panel#199] [Q&A] [answered]
Hello !
Any chance to have a foobar2000 v2.0 64 bits version in the near future ?
Regards

**Comment by TheQwertiest** [comment, marked as answer]
Not in the near future, but it will be implemented as a part of v2. You can track v2 progress here https://github.com/TheQwertiest/foo_spider_monkey_panel/issues/213
